docs/guide/concepts.md

```diff
@@ -2,11 +2,11 @@
 sidebarDepth: 2
 ---
 
-# Dva 概念
+# Dva concept
 
-## 数据流向
+## Data Flow
 
-数据的改变发生通常是通过用户交互行为或者浏览器行为（如路由跳转等）触发的，当此类行为会改变数据的时候可以通过 `dispatch` 发起一个 action，如果是同步行为会直接通过 `Reducers` 改变 `State` ，如果是异步行为（副作用）会先触发 `Effects` 然后流向 `Reducers` 最终改变 `State`，所以在 dva 中，数据流向非常清晰简明，并且思路基本跟开源社区保持一致（也是来自于开源社区）。
+Data changes usually occur through user interaction or browser behavior (such as route jumps). When such behavior changes data, an action can be initiated via `dispatch`. If it is synchronous, it will pass directly. `Reducers` change `State`, if it is asynchronous behavior (side effects) will trigger `Effects` and then flow to `Reducers` and finally change `State`, so in dva, the data flow is very clear and concise, and the idea is basically consistent with the open source community. (also from the open source community).
```

Comment:
Data changes usually occur through user interaction or browser behavior (such as ~~route jumps~~ url redirects). When such behavior changes data, an action can be initiated via `dispatch`. If it is synchronous, it will pass directly and `Reducers` change `State`. If it is asynchronous behavior (side effects), `Effects` will be triggered, then data flows to `Reducers`, and finally `State` will be updated. So in dva, the data flow is very clear. The principles are basically consistent with other open source community.

Reply:
"The basic principles are consistent with other open source community..."
Should it be "...open source community **practices**"?